internal/pkg/v2/dto/core/data/event.go

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package data
+
+import (
+	dtoBase "github.com/edgexfoundry/edgex-go/internal/pkg/v2/dto/common/base"
+	model "github.com/edgexfoundry/edgex-go/internal/pkg/v2/model/core/data"
+)
+
+// AddEventRequest defines the Request Content for POST event DTO. This object and its properties correspond to the
+// AddEventRequest object in the APIv2 specification.
+type AddEventRequest struct {
+	dtoBase.Request `json:",inline"`
+	Device          string          `json:"device"`
+	Origin          string          `json:"origin,omitempty" codec:"origin,omitempty"`
+	Readings        []model.Reading `json:"readings,omitempty" codec:"readings,omitempty"`
```

Comment:
Why is DTO referencing models? The purpose of the move to DTOs is to not leak our models. Per the V2 API Spec we should have a BaseReading DTO referenced here.

Reply:
Change has been made to add the `Reading` struct to [Reading DTO file ](https://github.com/edgexfoundry/edgex-go/blob/a1b7b6c8e8fae61bece8c24517cf07fa7b5af9d4/internal/pkg/v2/dto/core/data/reading.go) based on the suggestions, thanks.